packages/views/organization_detail.jade

```diff
@@ -1,23 +1,48 @@
 template(name="organizationDetail")
   .organization-detail
-    .container
-      .view-header
-        h1= organization.name
-        if currentUser
-          unless userIsMember
-            a.btn.btn-default.header-btn.join-organization
-              span Join
-              i.fa.fa-users
-      +paragraphText text=organization.description
-      .row
-        if memberCount
-          .col-sm-7
-            h2 Members
-              span.member-count (#{memberCount})
-            table.table.member-table
+    section
+      .container
+        if editingName
+          form.edit-name
+            input#organization-name.form-control(type="string" name="name" value=organization.name)
+            .actions.middle
+              button.btn-plain.save.save-name-button(type="submit")
+                i.fa.fa-check-circle
+              button.btn-plain.cancel.cancel-name-button(type="button")
+                i.fa.fa-times-circle
+        else
+          h1= organization.name
+          if userIsAdmin
+            button.btn-link.edit-inline.edit-name-button
+              span Edit
+              i.fa.fa-edit
+    section.organization-description-wrap.gray
+      .container.organization-description
+        if editingDescription
+          textarea#organization-description.form-control(type="string" name="description")= organization.description
+          .actions.top
+            button.btn-plain.save.save-description-button(type="submit")
+              i.fa.fa-check-circle
+            button.btn-plain.cancel.cancel-description-button(type="button")
+              i.fa.fa-times-circle
+        else
+          +paragraphText text=organization.description
+          if userIsAdmin
+            button.btn-link.edit-inline.edit-description-button
+              span Edit
+              i.fa.fa-edit
+      .container
+        .row
+          .panel.panel-default
+            .panel-heading Members
+            .panel-body
+              unless userIsMember
+                a.btn.btn-default.join-organization Join
+            table.table
               thead
                 tr
                   th Name
+                  th Email
```

Comment:
I thought we decided to remove this column due to the complications of only publishing emails that users consented to make public.